modules/text/include/opencv2/text/ocr.hpp

```diff
@@ -274,6 +296,8 @@ types.
  */
 CV_EXPORTS Ptr<OCRHMMDecoder::ClassifierCallback> loadOCRHMMClassifierNM(const std::string& filename);
 
+CV_EXPORTS_W Ptr<OCRHMMDecoder::ClassifierCallback> loadOCRHMMClassifierNM(const char* filename);
```

Comment:
I think, we should just change the original function parameter from "const std::string&" to "const String&"